avro/message.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "message",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'messages'",
+  "fields": [
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "name": "MessageCrudOperation",
+        "symbols": [
+          "CREATE",
+          "UPDATE",
+          "DELETE"
+        ]
+      }
+    },
+    {
+      "name": "id",
+      "type": "string",
+      "default": "undefined"
+    },
+    {
+      "name": "senderServiceId",
+      "type": "string",
+      "default": "undefined"
+    },
+    {
+      "name": "senderUserId",
+      "type": "string",
+      "default": "undefined"
+    },
+    {
+      "name": "timeToLiveSeconds",
+      "type": "int",
+      "default": 3600
+    },
+    {
+      "name": "createdAt",
+      "type": "int",
+      "logicalType": "time-millis",
+      "default": 0
+    },
+    {
+      "name": "isPending",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "content",
```

Comment:
pdnd asked us to provide data flattened (when possible). Maybe we can flatten content and payment data fields directly on the root.